### PR TITLE
Extend VerifySsl option params to allow set host and peer verification separately

### PR DESCRIPTION
**Pull request by Liastre**

### CODE REVIEW

File: cpr/ssl_options.cpp [hunk 1]
```diff
@@ -1,11 +1,18 @@
 #include "cpr/ssl_options.h"
 
-namespace cpr {
+using namespace cpr;
```
Comment (COM8) [inline cpr/ssl_options.cpp:3]: Do not use `using` here.

Comment (Liastre) [reply cpr/ssl_options.cpp:3]: Why not, it is good practice to use `using namespace` within translation unit scope. Using the whole scope is unnecessary nesting.

Comment (COM8) [reply cpr/ssl_options.cpp:3]: Yes, but it's against the coding "style" of this library and in my eyes better to read.
Also it does not spam so much functions into the auto complete if you do not use `using namespace XYZ`.

**Comment by COM8** [comment]
I'm closing this, since there does not seem to be any activity here any more.
Feel free to open a new PR.

**Pull request closed without merging**